### DISCUSSION TITLE
BoundaryAttack ValueError: The shape of mean and the standard deviation must be identical.

### DISCUSSION THREAD

**Discussion by HuannnXu** [Trusted-AI/adversarial-robustness-toolbox#1893] [Q&A] [answered]
When I tried the example of ART Boundary Attack, I got the ValueError: The shape of mean and the standard deviation must be identical when the program execute classifiers.predict()

**Comment by HuannnXu** [comment, marked as answer]
I worked it out by "classifier = KerasClassifier(clip_values=(0, 255), model=model, preprocessing=(mean_imagenet, np.ones([224, 224, 3])))" instead of classifier = KerasClassifier(clip_values=(0, 255), model=model, preprocessing=(mean_imagenet, 1))

**Reply by beat-buesser** [reply]
Hi @HuannnXu That's great!
